puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/katello-master-release.yaml

```diff
@@ -4,6 +4,12 @@
     sandbox: false
     triggers:
       - github
+      - build-result:
```

Comment:
This job already triggers a build via the groovy script

Reply:
Where?

Reply:
https://github.com/theforeman/foreman-infra/blob/4eddf0759e4690b9dfa7b4f4cf339dae07886b74/puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/rubygemRPMNightlyRelease.groovy#L50-L56

Reply:
It does mean that every nightly triggers it while I think it's only needed on a Foreman RPM, right?

Reply:
Correct! But given we use a generic job instead of a specific job, I can't decipher the Foreman specific build

Reply:
It looks like the parametrized trigger plugin does allow to filter on parameters:
> Restrict matrix execution to a subset

Reply:
I don't think that does what you think it does from looking around.